src/Microsoft.OData.Core/Uri/ODataUriConversionUtils.cs

```diff
@@ -61,6 +61,22 @@ internal static string ConvertToUriEnumLiteral(ODataEnumValue value, ODataVersio
             return string.Format(CultureInfo.InvariantCulture, "{0}'{1}'", value.TypeName, value.Value);
         }
 
+        /// <summary>
+        /// Converts the given string <paramref name="value"/> to an ODataResourceValue and returns it.
+        /// Tries in both JSON light and Verbose JSON.

```

Comment:
We should remove support for Verbose JSON (and certainly not advertise it in documentation)